CHANGELOG.md

```diff
@@ -189,6 +189,14 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 - Support `overrides` field in Prettier configuration files when migrating from Prettier.
   Contributed by @Conaclos
 
+- Support passing a file path to the `--config-path` flag or the `BIOME_CONFIG_PATH` environment variable.
+
+  Now you can pass a `.json`/`.jsonc` file path with any filename to the `--config-path` flag or the
+  `BIOME_CONFIG_PATH` environment variable. This will disable the configuration auto-resolution and Biome
+  will try to read the configuration from the said file path ([#2265](https://github.com/biomejs/biome/issues/2265)).
+  
+  Contributed by @Sec-ant
```

Comment:
``````suggestion
  will try to read the configuration from the said file path ([#2265](https://github.com/biomejs/biome/issues/2265)).
  ```shell
  biome format --config-path=../biome.json ./src
  ```
  Contributed by @Sec-ant
``````

Great changelog! I thought we could add an example :)